incrementalDelivery/v0.1/incrementalDelivery-v0.1.md

```diff
@@ -0,0 +1,280 @@
+# incrementalDelivery v0.1
+
+**Source: [Working Draft - September 2024](https://github.com/graphql/graphql-wg/blob/f22ea7748c6ebdf88fdbf770a8d9e41984ebd429/rfcs/DeferStream.md)**
+
+# Introduction
+
+One of the disadvantages of GraphQL’s request/response model is that applications which retrieve large datasets may suffer from latency. However not all requested data may be of equal importance, and in some use cases it may be possible for applications to act on a subset of the requested data.  
+
+Today applications which seek to ensure that data of high importance can be retrieved quickly have a few options:
+
+* Splitting queries
+* Prefetching
+
+However each of these solutions imposes some undesirable trade-offs on applications.
+
+## Splitting queries
+
+Given a query where some fields are expensive and non-essential, non-essential fields can be fetched in a separate query issued after an initial query. In the context of lists, this may mean fetching only the first few items in a list and issuing a follow-up pagination request for additional items. By separating the requests based on data priority, we have achieved a faster delivery time to essential data. However, query splitting imposes several **trade-offs** on applications:
+
+**Increased latency for lower priority fields,** as now these fields have to wait for the original query to complete and have added network round trips
+**Client resource contention.** Issuing multiple requests can increase contention on scarce resources like battery and antenna. 
+**Increased cost.** Processing additional requests can increase server costs by putting pressure on both the middle tier and the data layer. For example a follow-up query may need to load a subset of the same data loaded by the initial query, putting additional pressure on data stores.
+
+## Prefetching
+This technique involves optimistically fetching data based on a prediction that a user will execute an action. Prefetching can be one of the most effective ways of reducing latency. However, a significant tradeoff with prefetching that a lot of applications cannot afford is **increased server cost due to incorrect predictions.** With an unsophisticated prefetch algorithm, applications can easily overfetch by a factor of 10 fold.
```

Comment:
The build process fails with

```
This technique involves optimistically fetching data based on a prediction that a user will execute an action. Prefetching can be one of the most effective ways of reducing latency. However, a significant tradeoff with prefetching that a lot of applications cannot afford is **increased server cost due to incorrect predictions.** With an unsophisticated prefetch algorithm, applications can easily overfetch by a factor of 10 fold.
^
Expected "\n", "\r" or "\r\n" but "T" found.
```

Looks like there needs to be a extra newline after each header?